src/Shared/Recruit.Vacancies.Client/Infrastructure/Reports/CsvBuilder.cs

```diff
@@ -45,23 +45,49 @@ private static CsvWriter GetCsvWriter(TextWriter streamWriter)
 
         private void WriteNoResults(CsvWriter csv, DateTime reportDate)
         {
+            WriteTotalHeader(csv, reportDate, 0, 2);
+        }
+
+        private void WriteTotalHeader(CsvWriter csv, DateTime reportDate, int total, int fieldCount)
+        {
+            csv.WriteField("PROTECT");
```

Comment:
What does PROTECT do?

Reply:
Nothing its just text.  Its on the v1 reports. I think its a data protection hint.